### ISSUE TITLE
SSD300 needs to be wrapped

### ISSUE THREAD

**Issue by vatai** [undertherain/benchmarker#177]
`do_pytorch.py` calls `loss = model(data, target)` (not `out = model(data); loss = criterion(out, target)` ). This is made possible using the stuff in `helpers_pytorch.py` which wrap the "raw" pytorch model, into something that does specific to the problem (i.e. applies the right loss).

This is not done for SSD300. It should be done.
